includes/classes/Indexable/Comment/Comment.php

```diff
@@ -85,6 +85,9 @@ public function format_args( $query_vars ) {
 		 */
 		if ( isset( $query_vars['offset'] ) ) {
 			$formatted_args['from'] = (int) $query_vars['offset'];
+			if ( empty( $query_vars['number'] ) ) {
```

Comment:
Awesome idea! What do you think about doing this when setting up $number variable right away? So that it is clear from the start what we are putting to start.

```
            $max_size = apply_filters( 'ep_max_results_window', 10000 );
            if ( isset( $query_vars['offset'] ) ) {
               $number = $max_size - $query_vars['offset'] ;
            } else {
               $number = $max_size 
            }
```

Reply:
@pschoffer I didn't want to affect the `$number` logic for this line below: https://github.com/Automattic/ElasticPress/blob/77907388b1b5473dc1070f8dce7e59bd2a0f43f7/includes/classes/Indexable/Comment/Comment.php#L97

Reply:
Good point!